Assets/NuGet/Editor/NugetHelper.cs

```diff
@@ -1169,8 +1277,11 @@ public static bool Install(NugetPackage package, bool refreshAssets = true)
                 {
                     string baseDirectory = Path.Combine(NugetConfigFile.RepositoryPath, string.Format("{0}.{1}", package.Id, package.Version));
 
+
+                    ReadOptions options = new ReadOptions();
```

Comment:
Were you hitting errors and needed this? Is the encoding of nuget packages documented as always being utf8?

Reply:
that's options was removed already.